.github/actions/version_checking/action.yml

```diff
@@ -0,0 +1,54 @@
+name: version_checking
+description: 'Сhecks the equality of version from code and the last repository tag'
+
+inputs:
+  module-name:
+    description: Lua module name as when using require() in tarantool
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup tarantool
+      uses: tarantool/setup-tarantool@v2
+      with:
+        tarantool-version: '2.10'
+
+    - name: Make rock
+      run: tarantoolctl rocks make
+      shell: bash
+
+    - name: Code version
+      run: |
+        VERSION=$(tarantool -e "print(rawget(require('${{ inputs.module-name }}'), '_VERSION'))")
```

Comment:
Why do you use rawget here? Does `-e` enable strict? For me it works ok.

Reply:
So, I had some problems due to `__index` [here](https://github.com/tarantool/queue/blob/master/queue/init.lua#L42).
I tried to get `_VERSION` value with `pcall`, but it did not work for `__index` when `_VERSION` is `nil`

Reply:
Well, `require('queue')._VERSION` should work even before box.cfg

Reply:
> So, I had some problems due to `__index` [here](https://github.com/tarantool/queue/blob/master/queue/init.lua#L42). I tried to get `_VERSION` value with `pcall`, but it did not work for `__index` when `_VERSION` is `nil`

Well, I think it's okay if `require('queue')._VERSION` would fail with error before our comparison if something is wrong.

Reply:
Done